modules/scheduler/gke-cluster/main.tf

```diff
@@ -36,6 +36,44 @@ locals {
 
   # multi networking needs enabled Dataplane v2
   derived_enable_dataplane_v2 = coalesce(var.enable_dataplane_v2, local.derived_enable_multi_networking)
+
+  rdma_networks     = [for network_info in var.additional_networks : network_info if strcontains(upper(network_info.nic_type), "RDMA")]
+  non_rdma_networks = [for network_info in var.additional_networks : network_info if !strcontains(upper(network_info.nic_type), "RDMA")]
+  apply_manifests_rdma_networks = flatten([
```

Comment:
> Why create these two lists?

When creating the NetworkParamSet and Network kubernetes manifests we need to refer to the RDMA subnetworks.

Having a separate list makes it easy to reason about the index within the list. As the names of the subnetworks are based on the index {prefix-0, prefix-1, …, prefix-7}.